src/components/package-search-list.js

```diff
@@ -28,6 +31,7 @@ export default function PackageSearchList({
             pathname: `/eholdings/packages/${item.content.id}`,
             search: location.search
           }}
+          active={isPackagePage && routeParams.id === (item.content && item.content.id)}
```

Comment:
One last thing I just thought of:

When we aren't viewing a preview pane, `routeParams.id` might be `undefined`. And if `item.content` is also `undefined` that would cause this to be `true`.

I think moving `item.content &&` to the beginning of this prop should suffice:

```
item.content && isPackagePage && routerParams.id === item.content.id
```